### DISCUSSION TITLE
Welcome to JAVA_OOP_PKA_Nhom_10 Discussions! bai tap giua ki

### DISCUSSION THREAD

**Discussion by Nguyen1976** [Nguyen1976/JAVA_OOP_PKA_Nhom_10#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nguyen1976** [comment]
Giải pháp bài 1 của bạn

**Reply by DucHuy74** [reply]
tôi cx đang tìm giải pháp

**Comment by Nguyen1976** [comment]
Bạn hãy tạo Interface có tên Signal và 2 class DiscreteSignal và ContinuousSignal nó sẽ implements Signal

**Reply by DucHuy74** [reply]
class ContinuousSignal co can tao them bbien buoc song k o hay chi can bien do

**Comment by Nguyen1976** [comment]
class ContinuousSignal chỉ để biến double lưu giá trị biên độ của tín hiệu liên tục

**Comment by Nguyen1976** [comment]
![image](https://github.com/user-attachments/assets/e0e5d4bd-c689-49e3-b3d7-762962c5d9cb)
 hãy chú ý đoạn này rất có thể bạn cũng sẽ nhầm

**Reply by DucHuy74** [reply]
oke ban
